docs/source/usersguide/install.rst

```diff
@@ -378,6 +378,17 @@ i.e.
 
     CXX=mpicxx cmake /path/to/openmc
 
+If you are doing depletion analysis, you may have to set
+
+.. code-block:: python
+
+    openmc.pool.USE_MULTIPROCESSING = False
```

Comment:
The object is `openmc.deplete.pool.USE_MULTIPROCESSING` - https://docs.openmc.org/en/stable/pythonapi/deplete.html#openmc.deplete.pool.USE_MULTIPROCESSING

And while I agree that we should make this more user facing, I don't think this section is right. Perhaps `docs/source/usersguide/troubleshoot.rst` or `docs/source/usersguide/depletion.rst`?